setup.py

```diff
@@ -19,7 +19,7 @@
     'waitress',
     'pyramid_scheduler',
     'simplejson',
-    'zope',
+    'zope', 'zope.sqlalchemy',
```

Comment:
This reverts 9a17fc13c72694c1c7a796833c4d7f485ed7508b. Zope.sqlalchemy should technically not be required anymore, so something somewhere is causing the dependency unintentionally.